shard.yml

```
@@ -4,7 +4,7 @@ version: 1.2.2
 authors:
   - Kim Burgess <kim@place.technology>
 
-crystal: 0.34.0
+crystal: ">= 0.34, < 2"
```

It would be good to be explicit with these so that they contain a full semver.
```suggestion
crystal: ">= 0.34.0, < 2.0.0"
```